addon/array/stateful.js

```diff
@@ -1,24 +1,23 @@
+import { compare } from '@ember/utils';
+import ArrayProxy from '@ember/array/proxy';
+import { makeArray, A } from '@ember/array';
+import { copy } from '@ember/object/internals';
+import { get, set, computed } from '@ember/object';
 import Ember from 'ember';
 import { fragmentDidDirty, fragmentDidReset } from '../states';
 
 /**
   @module ember-data-model-fragments
 */
 
-const get = Ember.get;
-const set = Ember.set;
-const computed = Ember.computed;
-const copy = Ember.copy;
-const makeArray = Ember.makeArray;
-
 /**
   A state-aware array that is tied to an attribute of a `DS.Model` instance.
 
   @class StatefulArray
   @namespace MF
   @extends Ember.ArrayProxy
 */
-const StatefulArray = Ember.ArrayProxy.extend(Ember.Copyable, {
+const StatefulArray = ArrayProxy.extend(Ember.Copyable, {
```

Comment:
What about this `Ember.Copyable`? Is that one of the ones that's not exposed?

Reply:
Looks like its private https://emberjs.com/api/ember/2.16/classes/Ember.Copyable